test/sprints.e2e-spec.ts

```diff
@@ -1570,6 +1570,22 @@ describe("Sprints Controller (e2e)", () => {
                 })
                 .expect(400);
 
+            // sprint id is for a voyage the team member isn't part of
+            await request(app.getHttpServer())
+                .post(sprintCheckinUrl)
+                .set("Cookie", accessToken)
```

Comment:
I got an error from running the e2e tests saying "error TS2552: Cannot find name 'accessToken'. Did you mean 'access_token'?". I made the suggested change and the test pass now.
```suggestion
                .set("Cookie", access_token)
```

Reply:
That was actually changed in a recent PR, I've figured out. Was driving me nuts trying to figure out why the tests had been passing before lol.